tests/snuba/api/endpoints/test_organization_sessions.py

```diff
@@ -1675,6 +1695,16 @@ def test_order_by_with_limit(self):
                 "totals": {"sum(session)": 0},
                 "series": {"sum(session)": [0]},
             },
+            {
+                "by": {"release": "1B", "session.status": "abnormal"},
+                "totals": {"sum(session)": 0},
+                "series": {"sum(session)": [0]},
+            },
+            {
+                "by": {"release": "1B", "session.status": "crashed"},
+                "totals": {"sum(session)": 1},
+                "series": {"sum(session)": [1]},
+            },
```

Comment:
I almost thought this test would be flakey, but it is actually deterministic because the order between `abnormal` and `crashed` is defined by `ensure_status_groups` 👍 . https://github.com/getsentry/sentry/blob/2d4d54896560045463f3c3155282a0e10411a321/src/sentry/release_health/metrics_sessions_v2.py#L189